content/docs/hooks-overview.md

```diff
@@ -1,24 +1,23 @@
 ---
 id: hooks-overview
-title: Hooks at a Glance
+title: एक नज़र में Hooks
 permalink: docs/hooks-overview.html
 next: hooks-state.html
 prev: hooks-intro.html
 ---
+React 16.8 में *Hooks* एक नया अतिरिक्त है। वे आपको class लिखने के बिना state और अन्य React प्रतिक्रिया सुविधाओं का उपयोग करने देते हैं।
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+Hooks [बैकवर्ड-कम्पेटिबल](/docs/hooks-intro.html#no-breaking-changes) है।. यह पृष्ठ अनुभवी React उपयोगकर्ताओं के लिए Hooks का अवलोकन प्रदान करता है। यह एक तेज अवलोकन है। यदि आप भ्रमित हो जाते हैं, तो इस तरह पीले बॉक्स की तलाश करे:
 
-Hooks are [backwards-compatible](/docs/hooks-intro.html#no-breaking-changes). This page provides an overview of Hooks for experienced React users. This is a fast-paced overview. If you get confused, look for a yellow box like this:
-
->Detailed Explanation
+>विस्तृत विवरण
 >
->Read the [Motivation](/docs/hooks-intro.html#motivation) to learn why we're introducing Hooks to React.
+>यह जानने के लिए [प्रेरणा](/docs/hooks-intro.html#motivation) पढ़ें कि हम Hooks का React में परिचय क्यों कर रहे हैं।
 
-**↑↑↑ Each section ends with a yellow box like this.** They link to detailed explanations.
+**↑↑↑ प्रत्येक खंड इस तरह से पीले बॉक्स के साथ समाप्त होता है.** वे विस्तृत स्पष्टीकरण से जुड़ते हैं.
 
 ## 📌 State Hook {#state-hook}
```

Comment:
`State Hook` => `स्टेट हुक`

Reply:
I think we should not translate keywords like state, hook or React, etc. I am taking the inspiration from the home page where you can see that keywords like `props`, `state` etc. are not translated.

Reply:
@gautam-pahuja actually we are not translating those words.
If its refering to React related things.
i think in this case it does refer to React related state and hook. lets keep it as it is